### DISCUSSION TITLE
ASA-BIOP Covariate Adjustment SWG - [Upcoming] Variance Estimation for G-computation with Binary Outcomes

### DISCUSSION THREAD

**Discussion by giscus** [carswg/carswg.github.io#22] [General]
# ASA-BIOP Covariate Adjustment SWG - [Upcoming] Variance Estimation for G-computation with Binary Outcomes

April 2024 Journal Club

https://carswg.github.io/journalclub/journalclub2.html

**Comment by JingyiLiu-Stat** [comment]
Nice
